Bottom: 161e7ab6c95a9867db3916c572b4e86070321915
Top:    9cab163eb8a44d6a393428f51d649cf3eafc14c7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 01:25:28 +0100

DEBUG DEBUG DEBUG DEBUG


---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index 679d973..f0d9297 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -381,8 +381,8 @@ draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
 
   /* draw vias */
   r_search (PCB->Data->via_tree, drawn_area, NULL, via_inlayer_callback, layer);
-  r_search (PCB->Data->pin_tree, drawn_area, NULL, hole_callback, NULL);
-  r_search (PCB->Data->via_tree, drawn_area, NULL, hole_callback, NULL);
+//  r_search (PCB->Data->pin_tree, drawn_area, NULL, hole_callback, NULL);
+//  r_search (PCB->Data->via_tree, drawn_area, NULL, hole_callback, NULL);
 }
 
 struct draw_funcs d_f = {
